leakcanary-android-core/src/main/java/leakcanary/internal/NotificationReceiver.kt

```diff
@@ -40,7 +40,7 @@ internal class NotificationReceiver : BroadcastReceiver() {
     ): PendingIntent {
       val broadcastIntent = Intent(context, NotificationReceiver::class.java)
       broadcastIntent.action = action.name
-      val flags = if (Build.VERSION.SDK_INT > 30) {
+      val flags = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Why `>= Build.VERSION_CODES.M` and not just `>= 30` ?

Reply:
Also, should we instead use `>= 23` ?

Reply:
Our Lord Jake has spoken: https://twitter.com/JakeWharton/status/1049421501559128065

I probably missed some spots though.

Reply:
Let's go with 23 if you don't mind.

Reply:
I disagree with that, but who am I to question? ;)
Amended to 23.

Reply:
Just to clarify, are there any other concerns? Thanks!

Reply:
Nope, thanks!